code/index.html

```diff
@@ -3,28 +3,110 @@
+        <input type="checkbox" value="romance" id="write" name="write">
+        <label for="romance">Romance</label>
+        <br>
+        <input type="checkbox" value="crime" id="write" name="write">
+        <label for="crime">Crime</label>
+        <br>
+        <input type="checkbox" value="non-fiction" id="write" name="write">
+        <label for="non-fiction">Non fiction</label>
+        <br>
+        <input type="checkbox" value="young-adults" id="write" name="write">
+        <label for="young-adults">Young adults</label>
+        <br>
+        <input type="checkbox" value="other" id="write" name="write">
+        <label for="other">Other</label>
+        <br>
+        <div class="button-wrapper">
+        <input type="submit" value="Submit" class="button">
+      </div>
+    </form>
+  </section>
+
 
     <section class="accordion">
       <!-- Accordion -->
-      <h1>FAQ</h1>
+      <h3>This is Rainy Nights</h3>
+
+      <div class="question" id="section1"><h4>About Rainy Nights</h4> <img src="./arrow.png" class="icon"> </div><div class="answer">
+        <p>Rainy Nights is an indipended publishing house based in New York. We sign publish both published and unpublished authors. As a house we mostly focus on works of fiction such as crime, romance and young adults novels but occationally we publish non fiction. </p></div>
 
-      <div class="question" id="section1">Section 1</div>
       <div class="answer">
-        <p>Lorem ipsum...</p>
+        <p>Rainy Nights is an indipended publishing house based in New York. We sign publish both published and unpublished authors. As a house we mostly focus on works of fiction such as crime, romance and young adults novels but occationally we publish non fiction. </p>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
+      <div class="question" id="section2"><h4>Publish with us</h4><img src="./arrow.png" class="icon"></div>
       <div class="answer">
-        <p>Lorem ipsum...</p>
+        <p>To publish with us you need to fill out the form to establish contact. After that we will read you script/s and decide if it the right fit for us. If we want to publish the novel we will begin negotiations regarding contracts. If not we will give notes on the novel and hope you come back.</p>
+      </div>
+
+      <div class="question" id="section3"><h4>What happens next?</h4><img src="./arrow.png" class="icon"></div>
+      <div class="answer">
+        <p>After signing the contract you are a part of Rainy nights Publishing. We will start with drafts and notes right away until we, together, feel that the novel is ready. The process of releasing a novel can be long and stressful but we will always be by your side. </p>
+      </div>
+
+      <div class="question" id="section4"><h4>Contact</h4><img src="./arrow.png" class="icon"></div>
+      <div class="answer">
+        <p>If you have any further questions call at : 0800 123 456 789 or email at contact@rainynights.com</p>
```

Comment:
I see you have made it into 4 different divs and gave them an individual id so js can summon them by the id. I like the way you name classes and divs, it is very straightforward and reusable!